pkg/manifest/installer.go

```diff
@@ -223,6 +234,7 @@ func ApplyAll(manifests name.ManifestMap, version version.Version, opts *Install
 	for c := range manifests {
 		logAndPrint("- %s", c)
 	}
+	fmt.Println()
```

Comment:
leftover debug?

Reply:
This empty line is used to separate the outputs in stdout:
```
This will install the default Istio profile into the cluster. Proceed? (y/N) y
Applying manifests for these components:
- CoreDNS
- Kiali
- NodeAgent
- Tracing
- Pilot
- Prometheus
- Injector
- IngressGateway
- Grafana
- Base
- Galley
- Telemetry
- EgressGateway
- Citadel
- PrometheusOperator
- Policy
- CertManager
- Cni
               <- This is the empty line I added.
Applying manifest for component Base
Finished applying manifest for component Base
...
```

Reply:
in that case, should be logAndPrint("\n") - we don't want to use fmt directly.

Reply:
Sure, fixed.